components/EventForm.tsx

```diff
@@ -21,13 +23,41 @@ export default function EventForm({ event, onSubmitClick, formAction }: EventFor
           title: '',
           page: '',
           type: '',
+          track: '',
```

Comment:
I think we should create a default "EVENT_FORM_DATA" constant or something and define it elsewhere (probably some sort of constant file or in a similarly appropriate place) rather than defining it here.

Reply:
> I think we should create a default "EVENT_FORM_DATA" constant or something and define it elsewhere (probably some sort of constant file or in a similarly appropriate place) rather than defining it here.

Can you elaborate on this? Not sure I understand what you're suggesting. Do you mean like a type ("EVENT_FORM_DATA") that contains some fields (title, page, type, track) and have a just one field in the event form component of that is of that new composite type?

Reply:
I'm suggesting that we create a constant called `EVENT_FORM_DATA` that holds the default values for an event form, for instance:
` const EVENT_FORM_DATA: ScheduledEvent = { ...defaultFieldsHere };`